src/PlaywrightSharp.Tests/Page/ClickTests.cs

```diff
@@ -252,20 +257,21 @@ public async Task ShouldWaitForHidden()
         [Fact]
         public async Task ShouldWaitForVisible()
         {
+            bool done = false;
             await Page.GoToAsync(TestConstants.ServerUrl + "/input/button.html");
             await Page.QuerySelectorEvaluateAsync("button", "b => b.style.display = 'none'");
 
-            var clickTask = Page.ClickAsync("button");
+            var clicked = Page.ClickAsync("button").ContinueWith(_ => done = true);
 
             for (int i = 0; i < 5; i++)
             {
                 await Page.EvaluateAsync("1");
             }
-            Assert.False(clickTask.IsCompleted);
+            Assert.False(done);

```

Comment:
Why did we have to change this? `clickTask.IsCompleted` should have worked here.

Reply:
we needed to `await` the task and this is how we assert task completions (by continuing and setting a flag to true)